src/vm/clarity.rs

```diff
@@ -71,6 +71,7 @@ pub enum Error {
     Interpreter(InterpreterError),
     BadTransaction(String),
     CostError(ExecutionCost, ExecutionCost),
+    PostConditionAbort(Option<Value>, AssetMap, Vec<StacksTransactionEvent>)
```

Comment:
Clarity doesn't know about post-conditions; it only knows about abort callbacks.  Should this be reflected in this variant?  i.e. maybe `TransactionAborted` instead of `PostConditionAbort`?

Reply:
Renamed to `AbortedByCallback` in cf052c232